distribution/lib/Standard/Base/0.0.0-dev/src/Data/Numbers.enso

```diff
@@ -719,6 +719,14 @@ type Float
         Locale.Value java_locale -> Panic.catch ParseException ((NumberFormat.getInstance java_locale).parse text) _->
             Error.throw (Number_Parse_Error.Error text)
 
+    ## PRIVATE
+       Maximum representable positive value, from the underlying Java double type.
+    max_value = Double.MAX_VALUE
```

Comment:
I'd define this constants in some `private` module rather than used `## PRIVATE` comment only.

Reply:
Done.